carols/in_the_bleak.ly

```diff
@@ -27,126 +27,165 @@ global= {
   \key f \major
 }
 
-sop = \context Voice = "sop"    {
+sop = \context Voice = "sop" \relative c''   {
```

Comment:
It's really hard for me to see what, if any, is the actual diff here, since every line is different due to linebreaks/barlines and the addition of `\relative`, and everything is a single commit. In the future, if it's not too hard, could you change style and content in separate commits? And if you remember, could you let me know what the actual content changes are here?

Reply:
Is the actual difference just in the timings/melismas to fit with the scansion of the new middle verses?

Reply:
that's a very legit request, will keep in mind!
yeah, looks like it's just Rosetti text and alignment/melismas for the verses, as well as system breaks